db/migration/V1__Initial_Schema.sql

```diff
@@ -0,0 +1,135 @@
+CREATE TABLE UserProfile (
+    ID BIGSERIAL PRIMARY KEY,
+    AuthID TEXT NOT NULL UNIQUE,
+    Name TEXT NOT NULL,
+    NickName TEXT NULL,
+    Email TEXT NOT NULL,
+    PhotoURL TEXT NULL,
+    LastUpdate TIMESTAMP NOT NULL
+);
+
+CREATE TABLE Channel (
+  ID BIGSERIAL PRIMARY KEY,
+  Name TEXT NOT NULL UNIQUE,
+  Description TEXT NULL,
+  LastUpdatedOn TIMESTAMP NOT NULL
+);
+
+CREATE TABLE Member (
+    ID BIGSERIAL PRIMARY KEY,
+    UserID BIGSERIAL NOT NULL REFERENCES UserProfile,
+    ChannelID BIGSERIAL NOT NULL REFERENCES Channel,
+    IsOwner BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
This probably should have been named just "owner" for compatibility with the Java object.

Java bean convention is that the accessor method for a `boolean x`, will be named `isX`. This would make the Java object's accessor `isIsOwner`.